source/slang/slang-options.cpp

```diff
@@ -972,9 +976,10 @@ struct OptionsParser
                             String name;
                             SLANG_RETURN_ON_FAIL(tryReadCommandLineArgument(sink, arg, &argCursor, argEnd, name));
 
-                            String slice = argStr.subString(1, index - 1);
+                            // Skip the initial -, up to the last -
+                            UnownedStringSlice passThruSlice(argStr.begin() + 1, argStr.begin() + index);
                             SlangPassThrough passThrough = SLANG_PASS_THROUGH_NONE;
-                            if (SLANG_SUCCEEDED(TypeTextUtil::findPassThrough(slice.getUnownedSlice(), passThrough)))
+                            if (SLANG_SUCCEEDED(TypeTextUtil::findPassThrough(passThruSlice, passThrough)))
```

Comment:
Unrelated to this change (since this change eliminates the `.getUnownedSlice()` call, rather than adding it): it is worth considering whether conversion from `String` to `UnownedStringSlice` should be implicit.